Lib/test/test_import/__init__.py

```diff
@@ -411,7 +411,7 @@ def test_case_sensitivity(self):
     def test_double_const(self):
         # Another brief digression to test the accuracy of manifest float
         # constants.
-        from test import double_const  # don't blink -- that *was* the test
+        from test.test_import.data import double_const  # don't blink -- that *was* the test
```

Comment:
I suggest to rephrase the funny comment to make it more useful. Something like:

"Importing double_const checks that float constants serialiazed by marshal as PYC files don't lose precision (SF bug 422177)."

It's unclear to me how a single import can create the PYC file and load from the PYC file. Usually, we create a module object, write the PYC file, but then use the already create module object: we don't throw away the module, and then deserialize the PYC file. Maybe the test should load the module twice?

Reply:
Done! :+1: